var/spack/repos/builtin/packages/py-torchgeo/package.py

```diff
@@ -33,6 +33,7 @@ class PyTorchgeo(PythonPackage):
     variant("docs", default=False, description="Install documentation dependencies")
     variant("style", default=False, description="Install style checking tools")
     variant("tests", default=False, description="Install testing tools")
+    variant("rebuild-me", default=False, description="CI test")
```

Comment:
IMO this should just do a full rebuild (or maybe that's what you were planning after getting some agreement on this?).